tests/test_hdfs.py

```diff
@@ -190,3 +236,16 @@ def has_running_executors():
     shakedown.wait_for(lambda: streaming_job_is_not_running(),
                        ignore_exceptions=False,
                        timeout_seconds=600)
+
+
+@pytest.mark.skipif(not utils.hdfs_enabled(), reason='HDFS_ENABLED is false')
+@pytest.mark.sanity
+def test_history():
+    job_args = ["--class", "org.apache.spark.examples.SparkPi",
+                "--conf", "spark.eventLog.enabled=true",
+                "--conf", "spark.eventLog.dir=hdfs://hdfs/history"]
```

Comment:
I'm assuming this crashes if the directory or history server isn't present?

Reply:
Yes, I tried this and the test failed.